.github/workflows/node-test.yml

```diff
@@ -20,6 +17,9 @@ jobs:
             - name: Install
               run: yarn install --frozen-lockfile
 
+            - name: Build
+              run: yarn build
```

Comment:
I guess this could technically be `yarn build-plugin` when just running the tests on a PR?

For the `node-publish.yml` workflow we def. want to build the entire thing.

Reply:
I think we want to make sure the app production build also succeeds on a PR, we could technically have a separate `build` workflow but this is probably sufficient for now?

Reply:
Yep, go for it.